system/Autoloader/Autoloader.php

```diff
@@ -295,45 +316,6 @@ protected function loadInNamespace(string $class)
 
 	//--------------------------------------------------------------------
 
-	/**
```

Comment:
Removing loadLegacy already has a commit in the 4.1 branch.

Reply:
Hmm maybe this should go to 4.1 then? What do you think?

Reply:
Yes. Retain this method for now.

Reply:
I see it now. I'm confused why that went to `4.1` because I think we want it in `4.0.5` as well. maybe that was before we decided to release both? I will try cherry-picking it along with this PR.

Reply:
Actually that's going to make a mess of merging the release notes. I will remove the test for now since it is failing and unnecessary.

Reply:
I think I see what happened. I must have opened **Autoloader.php** while I was still on `4.1` and accidentally started working off that. I will restore it and apply to the commit to the correct base version, which should rebase without too much effort into `4.1`.